ghga_transpiler/cli.py

```diff
@@ -13,3 +13,55 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+""" CLI-specific wrappers around core functions."""
+import json
+from pathlib import Path
+from typing import Optional
+
+import typer
+from typing_extensions import Annotated
+
+HERE = Path(__file__).parent.resolve()
+DEFAULT_OUTPUT_FILE = HERE / "transpiled_metadata.yaml"
+
+cli = typer.Typer()
+
+
+def convert_workbook(filename):
+    """Function to run steps for conversion
+
+    This is a place holder
+
+    """
+
+    return filename
+
+
+@cli.command()
+def cli_main(
+    spread_sheet: Annotated[
+        Path,
+        typer.Option(
+            ...,
+            exists=True,
+            help="The path to input file",
+            dir_okay=False,
+            readable=True,
+        ),
+    ],
+    output_file: Annotated[
+        Optional[Path], typer.Option(..., help="The path to output file.")
+    ] = None,
+):
```

Comment:
```suggestion
def cli_main(
    spread_sheet: Path = typer.Argument(
        ...,
        exists=True,
        help="The path to input file",
        dir_okay=False,
        readable=True,
    ),
    output_file: Optional[Path] = typer.Argument(None, help="The path to output file."),
):
```

We're stuck with typer 0.7.0 for now